web/src/scenes/team/Credentials/index.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck FIXME: temporary disable typescript checking
```

Comment:
Seems like we don't use this component. @michaelfurutin Can you confirm? 

@maxpetretta Should we just remove it?

Reply:
yes, this component is not used

Reply:
Will remove on my end